pkgs/development/tools/language-servers/ruff-lsp/default.nix

```diff
@@ -53,6 +53,11 @@ buildPythonPackage rec {
   makeWrapperArgs = [
     # prefer ruff from user's PATH, that's usually desired behavior
     "--suffix PATH : ${lib.makeBinPath [ ruff ]}"
+
+    # Unset ambient PYTHONPATH in the wrapper, so Poetry only ever runs with
```

Comment:
I think poetry is wrong here

Reply:
Oh yes, I just copypasted the comment from its file lol.